test/WebSites/TagHelpersWebSite/Views/Catalog_CacheTagHelper/_SplashPartial.cshtml

```diff
@@ -0,0 +1,5 @@
+﻿<partial-title>Listing items</partial-title>
+<cache asp-vary-by="@ViewBag.Category" data-content="some-content">
```

Comment:
nit: `data-content` doesn't show up in the response and isn't used by the helper.  what is it for?